pkg/proxy/apis/config/validation/validation.go

```diff
@@ -293,9 +293,16 @@ func validateHostPort(input string, fldPath *field.Path) field.ErrorList {
 	return allErrs
 }
 
-func validateKubeProxyNodePortAddress(nodePortAddresses []string, fldPath *field.Path) field.ErrorList {
+func validateKubeProxyNodePortAddress(nodePortAddresses []string, nodePortAddressesPrimary bool, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 
+	if nodePortAddressesPrimary {
+		if len(nodePortAddresses) > 0 {
+			allErrs = append(allErrs, field.Invalid(fldPath, nodePortAddresses, "must not be set when nodePortAddressesPrimary is true"))
```

Comment:
Shouldn't the error bias in favor of the _old_ flag.  So `nodePortAddressesPrimary may not be set when nodePortAddresses is specified` or something

Reply:
I was thinking in terms of "which mistake is the user more likely to make". It seems like if someone said `--nodeport-addresses-primary` that they probably wanted that, so specifying `--nodeport-addresses ...` as well would be a mistake.

(But if we're switching to `--nodeport-addresses primary` then you can't specify "both" anyway.)